### DISCUSSION TITLE
v2/zh/guide/layout/breadcrumb

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#416] [Announcements]
# v2/zh/guide/layout/breadcrumb

vuepress-theme-hope 添加了开箱即用的路径导航支持，你可以通过 breadcrumb 字段控制它 无需任何额外配置，一个和主题色相符合的路径导航会显示在页面内容最上方，帮助阅读者理解文档结构。 你也可以通过 breadcrumbIcon 字段

https://vuepress-theme-hope.github.io/v2/zh/guide/layout/breadcrumb.html

**Comment by VivaAlexYu** [comment]
150版本发现路径导航已发生改变，表现为：没有README.md的文件也会创建路径导航，请问怎么调整为之前的逻辑，即路径下没有README.md 则忽略该路径导航。

**Reply by Mister-Hope** [reply]
https://theme-hope.vuejs.press/zh/guide/layout/catalog.html
